app/models/solidus_paypal_braintree/gateway.rb

```diff
@@ -4,6 +4,9 @@ module SolidusPaypalBraintree
   class Gateway < ::Spree::PaymentMethod
     include RequestProtection
 
+    class Error < StandardError; end
```

Comment:
Is the `Error` class needed? `TokenGenerationDisabledError` could inherit from `StandardError`

Reply:
I'm not sure how many errors are planned or could be used. We could but sometimes its nice to just rescue `Gateway::Error` which catches all errors within the class

Reply:
Are there any cons in adding this `Error` class later, only if needed?

Reply:
Nope, I can do that